### PR TITLE
Update github workflow

### PR DESCRIPTION
**Pull request by LaRiffle**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+setuptools==42.0.0
+wheel==0.33.6
+pytest==5.2.1
+twine==3.1.0
+flake8==3.7.9
+black
```
[the remark this answers is not in the document]
Comment (LaRiffle) [reply requirements-dev.txt]: Ouais but I could manage to get it right, how do you do this usually?

[the remark this answers is not in the document]
Comment (Jasopaum) [reply requirements-dev.txt]: You could set the black version to the one on your machine

[the remark this answers is not in the document]
Comment (LaRiffle) [reply requirements-dev.txt]: yeah this is `pip freeze`

[the remark this answers is not in the document]
Comment (LaRiffle) [reply requirements-dev.txt]: :)

**Pull request merged**